openmessaging-format.md

```diff
@@ -0,0 +1,130 @@
+# OpenMessaging Event Format for CloudEvents - Version 0.1
+
+## Abstract
+
+The OpenMessaging Format for CloudEvents defines how events are expressed in
+OpenMessaging.
```

Comment:
can you turn this into a link so people can easily find OpenMessaging?

Reply:
Yeah, we will add a link to OpenMessaging.